android/anc/src/test/java/org/smartregister/fhircore/anc/data/family/FamilyRepositoryTest.kt

```diff
@@ -113,6 +114,35 @@ class FamilyRepositoryTest : RobolectricTest() {
     coVerify { resourceMapperExtended.saveParsedResource(any(), any(), any(), null) }
   }
 
+  @Test
+  fun updateProcessFamilyMemberShouldExtractEntities() = runBlockingTest {
```

Comment:
Rename to `updateProcessFamilyMemberShouldCallSaveParsedResourceWithEditBooleanAsTrue`